config/init/upstart/Makefile.am

```diff
@@ -1,4 +1,4 @@
-EXTRA_DIST = lxc.conf lxc-instance.conf lxc-net.conf.in
+EXTRA_DIST = lxc.conf.in lxc-instance.conf lxc-net.conf.in
```

Comment:
This is `lxc.conf` without the `.in`-prefix on purpose; it doesn't need to be processed before.